source/_docs/guides/build-tools/01-introduction.md

```diff
@@ -0,0 +1,120 @@
+          <p class="pantheon-official"></p>
+        </div>
+      </div>
+      <p class="topic-info__description"><a href="/docs/composer/">Composer</a> is a PHP dependency manager that provides an alternative, more modern way to manage the external code used by a project. For example, Composer may be used to install the plugins, modules and themes used by a Drupal or WordPress site.</p>
+    </div>
+  </div>
+</div>
+
+<div class="enablement" markdown="1">
+  <h4 class="info" markdown="1">[Automation Training](https://pantheon.io/agencies/learn-pantheon?docs){.external}</h4>
+  Master Composer concepts with help from our experts. Pantheon delivers custom workshops to help development teams master the platform and improve internal DevOps.
+</div>
+
+## Artifact Deployment
+Only files unique to the project are tracked as part of the project's main "source" repository on GitHub, which requires an abstraction layer to compile dependencies and deploy an entire "artifact" to the site repository on Pantheon. The abstraction layer is facilitated by CirlceCI in the Pantheon maintained examples, but the principles are the same for other continuous integration service providers.
+
+Composer is used to fetch dependencies declared by the project as part of a CircleCI build step. This ensures that the final composed build results are installed on Pantheon:
+
+<p class="text-center" markdown="1">![Artifact Deployment](/source/docs/assets/images/artifact-deployment.png)</p>
+
+## Before You Begin
+
+1. Install [Composer](https://getcomposer.org){.external}
+2. Install the most recent release of [Terminus](/docs/terminus/):
+
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-installer">Copy</button>
+      <figure><pre id="terminus-installer"><code class="command bash" data-lang="bash">curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar install</code></pre></figure>
+    </div>
+
+3. [Generate a Machine Token](https://dashboard.pantheon.io/machine-token/create){.external}, then authenticate Terminus:
+
+      <div class="copy-snippet">
+        <button class="btn btn-default btn-clippy" data-clipboard-target="#mac-mt-auth">Copy</button>
+        <figure><pre id="mac-mt-auth"><code class="command bash" data-lang="bash">terminus auth:login --machine-token=&lsaquo;machine-token&rsaquo;</code></pre></figure>
+      </div>
+
+4. Install the [Terminus Composer Plugin](https://github.com/pantheon-systems/terminus-composer-plugin){.external}:
+
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#composer-plugin">Copy</button>
+      <figure><pre id="composer-plugin"><code class="command bash" data-lang="bash">curl https://github.com/pantheon-systems/terminus-composer-plugin/archive/1.0.0.tar.gz -L | tar -C ~/.terminus/plugins -xvz</code></pre></figure>
```

Comment:
Installing with `curl | tar` is not a good idea. This method contains the version number of the tool to download and install in the URL. Doing it this way would require us to update the documentation every time a new release is made; otherwise, folks would get the old version. There is still a risk that the download instructions could be copied e.g. into user notes, and external blog post, etc., and there is no way to clean up these stale references. We should instead document installing with `composer create-project`, which will always install the most recent version.